docs/user-guide/migrating-to-6.0.0.md

```diff
@@ -111,6 +112,22 @@ If you use a config file located outside of a local project (with the `--config`
 
 **Related issue(s):** [eslint/eslint#9687](https://github.com/eslint/eslint/issues/9687), [eslint/espree#384](https://github.com/eslint/espree/issues/384)
 
+## <a name="rule-config-validating"></a> Rule configuration are validated more strictly
+
+To catch config errors earlier, ESLint v6 will report a linting error if you are trying to configure a non-existent rule.
```

Comment:
```suggestion
To catch config errors earlier, ESLint v6 will report a linting error if you are trying to configuring a non-existent rule.
```

Reply:
I think the original wording ("to configure") is correct here.

Reply:
Does the semantic here mean "attempt to do something"?

Reply:
yes. try to do sth.

Reply:
That’s right! This can be rewritten as “...if you are attempting to configure a non-existent rule.”

Reply:
So this should be `try doing sth` or `attempt to do sth`, not `try to do sth`.

Reply:
I think it’s fine as it is. “Try doing” would be grammatically incorrect in this case.

Reply:
OK.